pkg/operator2/route.go

```diff
@@ -58,14 +58,17 @@ func (c *authOperator) handleRoute(ingress *configv1.Ingress) (*routev1.Route, *
 }
 
 func defaultRoute(ingress *configv1.Ingress) *routev1.Route {
+	var weightVal int32 = 100
```

Comment:
Add a comment noting that this is emulating server side defaulting with a link to https://github.com/openshift/openshift-apiserver/blob/a48daecd8d8f22de33be3c7ea8127672f84d46e0/pkg/route/apis/route/v1/defaults.go

Add a TODO to replace this with server side apply.  We have the same problem with services IIRC.